editor.planx.uk/src/@planx/components/List/Public/List.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+
+import Wrapper from "../../fixtures/Wrapper";
+import { SCHEMAS } from "../Editor";
+import Editor from "../Editor";
+import ListComponent from "../Public";
+import Public from "./";
+
+const meta = {
+  title: "PlanX Components/List",
+  component: ListComponent,
+} satisfies Meta<typeof ListComponent>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+const schemaFinder = (name: string) => {
+  const schemaObj = SCHEMAS.find((schema) => schema.name === name)?.schema;
+  return schemaObj || SCHEMAS[0].schema;
+};
+
+export const Basic: Story = {
+  args: {
+    title: "List",
+    description: "Add information on the removed residential units",
```

Comment:
nit: "List" will already be the top-level Storybook page title, so we'd normally try to make the actual component title more realistic/match how it's actually implemented in content - eg: 
```
title: "Describe any residential units that are being removed or lost as part of the development",
description: "Add one or many units below"
```

It can also be helpful to set mock help text prop(s) on at least one example so it's clear where the "More information" link would appear !

Reply:
Great, I'll add it before merging